HANDSON1.md

```diff
@@ -31,7 +31,7 @@ You interact with Smart Test using a command line tool called `launchable`.
 You can install it with pip:
 
 ```
-pip install --user --upgrade launchable~=1.0
```

Comment:
If set `--user` option, the target command will be installed under the `~/local/bin` directory, basically.
https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-user

I don't think it's common to add `~/.local/bin` to PATH. 

WDYT??